minichef/tests/test_simple_harvest.py

```diff
@@ -17,18 +17,20 @@ def test_simple_harvest(
     chef,
     synapseToken,
     pid,
-    usdt
+    usdt,
+    dai
 ):
     ## deposit to the vault after approving
     startingWhale = poolToken.balanceOf(whale)
     poolToken.approve(vault, 2 ** 256 - 1, {"from": whale})
-    vault.deposit(amount, {"from": whale})
+    vault.deposit(poolToken.balanceOf(whale), {"from": whale})
```

Comment:
Amount should be a fixed value like it was previously compared to dynamic one that is now.
That way we can ourselves determine what amount is large enough for our tests.
If this whale decided to leave it's position almost completely, and was testing with let's say 5lp tokens, the tests wouldn't be failing but they may not be exactly right since we are also testing if the strat can handle big enough liquidity